### PR TITLE
Documentation update 2

### PR DESCRIPTION
**Pull request by agenius-jasmine**

### CODE REVIEW

**Comment by agenius-jasmine** [comment]
not sure why there are still test failures now despite correcting them only a couple of commits ago... if you runt he tests in the test_plotDispersion.R there are no errors.

I will investigate - but commenting here so I don't forget in the new year:
```
── Test failures ───────────────────────────────────────────────────────────────────────────────── testthat ────

> library(testthat)
> library(DGEobj.plots)
> 
> test_check("DGEobj.plots")
Loading required package: limma
Loading required package: DGEobj
Loading required package: DGEobj.utils
Loading required package: dplyr

Attaching package: 'dplyr'

The following object is masked from 'package:testthat':

    matches

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

trying URL 'http://ftp.ncbi.nlm.nih.gov/geo/series/GSE120nnn/GSE120804/suppl/GSE120804_qc_metrics.txt.gz'
Content type 'application/x-gzip' length 22175 bytes (21 KB)
==================================================
downloaded 21 KB

══ Failed tests ════════════════════════════════════════════════════════════════
── Error (test_plotDispersion.R:8:5): plotDispersion.R: plotDispersion() ───────
Error in `fitFDistRobustly(var, df1 = df, covariate = covariate, winsor.tail.p = winsor.tail.p)`: statmod package required but is not installed
Backtrace:
    █
 1. ├─DGEobj.plots::plotDispersion(dgeObj = t_obj1) test_plotDispersion.R:8:4
 2. │ └─`%>%`(...)
 3. ├─edgeR::estimateDisp(...)
 4. └─edgeR::estimateDisp.DGEList(...)
 5.   ├─edgeR::estimateDisp(...)
 6.   └─edgeR::estimateDisp.default(...)
 7.     └─limma::squeezeVar(...)
 8.       └─limma::fitFDistRobustly(...)

[ FAIL 1 | WARN 0 | SKIP 0 | PASS 1087 ]
Error: Test failures
Execution halted

1 error x | 0 warnings ✓ | 1 note x
```

File: R/QCplots.R [hunk 1]
```diff
@@ -6,25 +6,24 @@
 #'
 #' Returns a list of QC plots for the metricNames provided as input.
 #'
-#' By default, horizontal reference lines are drawn  at the median and +/- n SDs based on the
+#' By default, horizontal reference lines are drawn at the median and +/- n SDs based on the
 #' hlineSD argument. These are statistical reference points, NOT pass/fail limits.
 #'
-#' @param DGEdata DGEobj(Required)
+#' @param DGEdata A DGEobj object with an an item of type "alignQC" (Required)
```
Comment (cb4ds) [inline R/QCplots.R:12]: double "an"

File: R/cdfPlot.R [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
 #'   (default = NULL; NULL disables, set to desired color to enable)
-#' @param insetX x-location for the inset plot (default = 0.15)
-#' @param insetY y-location for the inset plot (default = 0.85)
-#' @param insetWidth width of the inset plot (default = 0.35)
+#' @param insetX x-location for the inset plot, only for ggplots (default = 0.15)
+#' @param insetY y-location for the inset plot, only for ggplots (default = 0.85)
+#' @param insetWidth width of the inset plot, only for ggplots (default = 0.35)
 #'
-#' @return A list of plots.
+#' @return A list of plots. 2 plots for canvasXpress and 3 plots for ggplot.
```
Comment (cb4ds) [inline R/cdfPlot.R:31]: change to `A list of plots -- 2 plots <rest of line> if plotType is 'canvasXpress' or 3 plots if plotType is 'ggplot'`

File: R/plotNorm.R [hunk 3]
```diff
@@ -6,16 +6,16 @@
 #' Either a box plot or density plot category can be chosen.
 #'
 #' Normalization is performed by edgeR::calcNormFactors. Note TMM is specifically tailored for count-based
-#' data.  Thus this function is only appropriate for count-based data.
+#' data.Thus this function is only appropriate for count-based data.
```
Comment (cb4ds) [inline R/plotNorm.R:9]: Add a space after the period.

File: R/volcanoPlot.R [hunk 4]
```diff
@@ -23,27 +23,27 @@
...
-#' @param sizeByIntensity If TRUE, creates a column to support sizeByIntensity. (default = TRUE)
+#' @param contrast Name of the contrast
+#' @param plotType Plot type must be canvasXpress or ggplot (default = canvasXpress)
+#' @param logRatioCol Name of the LogRatio column (default = "logFC")
+#' @param logIntCol Name of the LogIntensity column (default = "AveExpr")
+#' @param pvalCol Name of the p-value or FDR column (default = "P.Value")
+#' @param xlab X axis label (default = NULL)
+#' @param ylab Y axis label (default = NULL)
+#' @param title Plot title (optional)
+#' @param pthreshold The p value threshold used to color points (default = 0.01)
```
Comment (cb4ds) [inline R/volcanoPlot.R:34]: dash between p and value "p-value"

File: R/volcanoPlot.R [hunk 5]
```diff
@@ -23,27 +23,27 @@
...
+#' @param geneNameCol geneName column in geneData from DGEobj. This column will be used to label significantly changed points (optional)
+#' @param pthresholdLine Color string to use a horizontal line at the p-threshold. (default = NULL (disabled))
```
Comment (cb4ds) [inline R/volcanoPlot.R:36]: "string" should be changed to value

File: R/volcanoPlot.R [hunk 6]
```diff
@@ -23,27 +23,27 @@
...
+#' @param sizeByIntensity To have the points sized by the LogInt column (default = TRUE) or by the group column (FALSE)
```
Comment (cb4ds) [inline R/volcanoPlot.R:37]: Rephrase to active: "Size points by the LogInt column...<rest>

**Review by cb4ds — CHANGES_REQUESTED** [review]

File: R/cdfPlot.R (hunk 2 above)
Comment (agenius-jasmine) [reply R/cdfPlot.R:31]: @cb4ds what do you mean by <rest of line> ?

File: R/QCplots.R [hunk 7]
```diff
@@ -9,9 +9,9 @@
 #' By default, horizontal reference lines are drawn at the median and +/- n SDs based on the
 #' hlineSD argument. These are statistical reference points, NOT pass/fail limits.
 #'
-#' @param DGEdata A DGEobj object with an an item of type "alignQC" (Required)
+#' @param DGEdata A DGEobj object with an item of type "alignQC" (required)
 #' @param metricNames A list of metrics to plot. Values must exist in column names of the item
-#' of type AlignQC. (Required)
+#' of type AlignQC. (required)
```
Comment (cb4ds) [inline R/QCplots.R:14]: It is kind of odd to say "required", we usually only point out optional parameters.  Is this consistent throughout the package?

**Review by cb4ds — APPROVED** [review]

File: R/QCplots.R (hunk 7 above)
Comment (agenius-jasmine) [reply R/QCplots.R:14]: ah  I see. I saw it in other functions so I kept it. I can create another PR to remove them all.

**Pull request merged**